src/options/view.rs

```diff
@@ -29,28 +29,24 @@ impl Mode {
     ///
     /// This is complicated a little by the fact that `--grid` and `--tree`
     /// can also combine with `--long`, so care has to be taken to use the
-    pub fn deduce<V: Vars>(matches: &MatchedFlags<'_>, vars: &V) -> Result<Self, OptionsError> {
-        let flag = matches.has_where_any(|f| f.matches(&flags::LONG) || f.matches(&flags::ONE_LINE)
-                                          || f.matches(&flags::GRID) || f.matches(&flags::TREE));
+    pub fn deduce<V: Vars>(matches: &Opts, vars: &V, stricness: bool) -> Result<Self, OptionsError> {
```

Comment:
typo (present in other functions’ signature):

```suggestion
    pub fn deduce<V: Vars>(matches: &Opts, vars: &V, strictness: bool) -> Result<Self, OptionsError> {
```